front/app/modules/commercial/smart_groups/containers/RulesGroupFormWithValidation/messages.ts

```diff
@@ -15,4 +15,16 @@ export default defineMessages({
     id: 'app.containers.AdminPage.Users.UsersGroup.rulesError',
     defaultMessage: 'Some conditions are incomplete',
   },
+  atLeastOneRuleError: {
+    id: 'app.containers.AdminPage.Users.UsersGroup.atLeastOneRuleError',
+    defaultMessage: 'Provide at least one rule',
+  },
+  titleFieldError: {
+    id: 'app.containers.AdminPage.Users.UsersGroup.titleFieldError',
+    defaultMessage: 'Provide group name for all languages',
+  },
```

Comment:
A detail really, but find the error id too generic here, could be any error. Rather `titleLanguagesMissingError` or something? (but also see previous related comment)